### DISCUSSION TITLE
Description

### DISCUSSION THREAD

**Discussion by vitalikbyrevich** [Digitalroot-Valheim/Digitalroot.Valheim.EpicLoot.Adventure.Bounties#2] [Q&A] [answered]
Good afternoon. Is it possible to make a slightly more detailed description? And then it is not clear what awards and for what
I changed the adventuredata file.json for adding things from Terraheim. Will your mod rewrite this file in its own way?

**Comment by Digitalroot** [comment]
The mod only changes the bounties. It does this via code and not editing the text files. 

I'll see about adding a list of all the bounties that are added.

**Reply by vitalikbyrevich** [reply]
That is, your mod will not affect my modified file?

**Reply by Digitalroot** [reply]
correct

**Comment by Digitalroot** [comment, marked as answer]
Ref: https://github.com/Digitalroot-Valheim/Digitalroot.Valheim.EpicLoot.Adventure.Bounties/blob/main/README.md

I do not want to go into more details then this, that would spoil the mystery. 

## Meadows
<table>
<tr><th>TargetID</th><th>Iron</th><th>Gold</th><th>Coins</th><th>Adds</th></tr>
<tr><td>Deer</td><td>1</td><td>0</td><td>10</td><td>:x:</td></tr>
<tr><td>Boar</td><td>1</td><td>0</td><td>10</td><td>:x:</td></tr>
<tr><td>Boar</td><td>1</td><td>0</td><td>10</td><td>:x:</td></tr>
<tr><td>Neck</td><td>1</td><td>0</td><td>10</td><td>:x:</td></tr>
<tr><td>Neck</td><td>1</td><td>0</td><td>10</td><td>:x:</td></tr>
<tr><td>Greyling</td><td>1</td><td>0</td><td>10</td><td>:x:</td></tr>
<tr><td>Greyling</td><td>1</td><td>0</td><td>10</td><td>:x:</td></tr>
<tr><td>Boar</td><td>2</td><td>0</td><td>15</td><td>:heavy_check_mark:</td></tr>
<tr><td>Neck</td><td>2</td><td>0</td><td>15</td><td>:heavy_check_mark:</td></tr>
<tr><td>Greydwarf_Shaman</td><td>2</td><td>1</td><td>25</td><td>:heavy_check_mark:</td></tr>
<tr><td>Greydwarf_Elite</td><td>10</td><td>3</td><td>100</td><td>:heavy_check_mark:</td></tr>
<tr><td>BlobElite</td><td>0</td><td>1</td><td>25</td><td>:x:</td></tr>
<tr><td>Skeleton_Poison</td><td>5</td><td>0</td><td>25</td><td>:x:</td></tr>
<tr><td>Skeleton_Poison</td><td>1</td><td>1</td><td>30</td><td>:heavy_check_mark:</td></tr>
<tr><td>Troll</td><td>0</td><td>1</td><td>50</td><td>:x:</td></tr>
<tr><td>Ghost</td><td>1</td><td>0</td><td>20</td><td>:x:</td></tr>
<tr><td>Ghost</td><td>1</td><td>0</td><td>20</td><td>:x:</td></tr>
<tr><td>Wraith</td><td>0</td><td>1</td><td>50</td><td>:x:</td></tr>
<tr><td>Lox</td><td>3</td><td>0</td><td>50</td><td>:x:</td></tr>
<tr><td>Eikthyr</td><td>3</td><td>2</td><td>100</td><td>:x:</td></tr>
<tr><td>Eikthyr</td><td>5</td><td>2</td><td>120</td><td>:heavy_check_mark:</td></tr>
<tr><td>Eikthyr</td><td>5</td><td>3</td><td>120</td><td>:heavy_check_mark:</td></tr>
<tr><td>Eikthyr</td><td>8</td><td>5</td><td>160</td><td>:heavy_check_mark:</td></tr>
</table>

## BlackForest
<table>
<tr><th>TargetID</th><th>Iron</th><th>Gold</th><th>Coins</th><th>Adds</th></tr>
<tr><td>Greydwarf</td><td>2</td><td>0</td><td>15</td><td>:x:</td></tr>
<tr><td>Greydwarf</td><td>2</td><td>0</td><td>15</td><td>:x:</td></tr>
<tr><td>Skeleton</td><td>2</td><td>0</td><td>15</td><td>:x:</td></tr>
<tr><td>Skeleton</td><td>2</td><td>0</td><td>15</td><td>:x:</td></tr>
<tr><td>Ghost</td><td>2</td><td>0</td><td>20</td><td>:x:</td></tr>
<tr><td>Greydwarf_Shaman</td><td>3</td><td>0</td><td>25</td><td>:x:</td></tr>
<tr><td>Skeleton</td><td>4</td><td>0</td><td>30</td><td>:heavy_check_mark:</td></tr>
<tr><td>Greydwarf_Elite</td><td>0</td><td>1</td><td>20</td><td>:x:</td></tr>
<tr><td>Troll</td><td>0</td><td>1</td><td>50</td><td>:x:</td></tr>
<tr><td>Greydwarf_Shaman</td><td>4</td><td>1</td><td>40</td><td>:heavy_check_mark:</td></tr>
<tr><td>gd_king</td><td>5</td><td>2</td><td>180</td><td>:x:</td></tr>
<tr><td>gd_king</td><td>7</td><td>2</td><td>200</td><td>:heavy_check_mark:</td></tr>
<tr><td>gd_king</td><td>9</td><td>3</td><td>220</td><td>:heavy_check_mark:</td></tr>
<tr><td>gd_king</td><td>9</td><td>3</td><td>220</td><td>:heavy_check_mark:</td></tr>
<tr><td>gd_king</td><td>12</td><td>3</td><td>250</td><td>:heavy_check_mark:</td></tr>
</table>

## Swamp
<table>
<tr><th>TargetID</th><th>Iron</th><th>Gold</th><th>Coins</th><th>Adds</th></tr>
<tr><td>Blob</td><td>3</td><td>0</td><td>25</td><td>:x:</td></tr>
<tr><td>BlobElite</td><td>4</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>BlobElite</td><td>5</td><td>0</td><td>35</td><td>:heavy_check_mark:</td></tr>
<tr><td>BlobElite</td><td>5</td><td>0</td><td>35</td><td>:heavy_check_mark:</td></tr>
<tr><td>BlobElite</td><td>5</td><td>0</td><td>35</td><td>:heavy_check_mark:</td></tr>
<tr><td>BlobElite</td><td>6</td><td>0</td><td>40</td><td>:heavy_check_mark:</td></tr>
<tr><td>BlobElite</td><td>5</td><td>0</td><td>35</td><td>:heavy_check_mark:</td></tr>
<tr><td>BlobElite</td><td>5</td><td>0</td><td>35</td><td>:heavy_check_mark:</td></tr>
<tr><td>BlobElite</td><td>5</td><td>0</td><td>35</td><td>:heavy_check_mark:</td></tr>
<tr><td>BlobElite</td><td>5</td><td>0</td><td>35</td><td>:heavy_check_mark:</td></tr>
<tr><td>Skeleton</td><td>3</td><td>0</td><td>20</td><td>:x:</td></tr>
<tr><td>Leech</td><td>3</td><td>0</td><td>20</td><td>:x:</td></tr>
<tr><td>Ghost</td><td>0</td><td>0</td><td>20</td><td>:x:</td></tr>
<tr><td>Draugr</td><td>3</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>Surtling</td><td>3</td><td>0</td><td>20</td><td>:x:</td></tr>
<tr><td>Surtling</td><td>4</td><td>0</td><td>30</td><td>:heavy_check_mark:</td></tr>
<tr><td>Wraith</td><td>0</td><td>1</td><td>50</td><td>:x:</td></tr>
<tr><td>Wraith</td><td>3</td><td>1</td><td>70</td><td>:heavy_check_mark:</td></tr>
<tr><td>Draugr_Elite</td><td>0</td><td>1</td><td>50</td><td>:x:</td></tr>
<tr><td>Draugr_Elite</td><td>5</td><td>1</td><td>80</td><td>:heavy_check_mark:</td></tr>
<tr><td>Draugr_Elite</td><td>5</td><td>1</td><td>80</td><td>:heavy_check_mark:</td></tr>
<tr><td>Draugr_Elite</td><td>5</td><td>1</td><td>80</td><td>:heavy_check_mark:</td></tr>
<tr><td>Draugr_Elite</td><td>5</td><td>1</td><td>80</td><td>:heavy_check_mark:</td></tr>
<tr><td>Skeleton</td><td>5</td><td>1</td><td>50</td><td>:heavy_check_mark:</td></tr>
<tr><td>Skeleton_Poison</td><td>5</td><td>1</td><td>60</td><td>:heavy_check_mark:</td></tr>
<tr><td>Bonemass</td><td>6</td><td>2</td><td>180</td><td>:x:</td></tr>
<tr><td>Eikthyr</td><td>5</td><td>1</td><td>120</td><td>:x:</td></tr>
<tr><td>gd_king</td><td>5</td><td>1</td><td>140</td><td>:x:</td></tr>
<tr><td>Bonemass</td><td>7</td><td>2</td><td>200</td><td>:heavy_check_mark:</td></tr>
<tr><td>Bonemass</td><td>8</td><td>2</td><td>210</td><td>:heavy_check_mark:</td></tr>
<tr><td>Bonemass</td><td>9</td><td>2</td><td>220</td><td>:heavy_check_mark:</td></tr>
<tr><td>Bonemass</td><td>8</td><td>4</td><td>210</td><td>:heavy_check_mark:</td></tr>
<tr><td>Bonemass</td><td>11</td><td>4</td><td>240</td><td>:heavy_check_mark:</td></tr>
</table>

## Mountain
<table>
<tr><th>TargetID</th><th>Iron</th><th>Gold</th><th>Coins</th><th>Adds</th></tr>
<tr><td>Wolf</td><td>4</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>Wolf</td><td>4</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>Hatchling</td><td>4</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>Hatchling</td><td>4</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>Skeleton</td><td>4</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>Skeleton</td><td>4</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>Surtling</td><td>4</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>Draugr_Ranged</td><td>4</td><td>0</td><td>30</td><td>:x:</td></tr>
<tr><td>Skeleton</td><td>5</td><td>0</td><td>40</td><td>:heavy_check_mark:</td></tr>
<tr><td>Fenring</td><td>0</td><td>2</td><td>50</td><td>:x:</td></tr>
<tr><td>Fenring</td><td>5</td><td>2</td><td>100</td><td>:heavy_check_mark:</td></tr>
<tr><td>Wolf</td><td>5</td><td>1</td><td>70</td><td>:heavy_check_mark:</td></tr>
<tr><td>StoneGolem</td><td>0</td><td>2</td><td>80</td><td>:x:</td></tr>
<tr><td>StoneGolem</td><td>7</td><td>3</td><td>160</td><td>:heavy_check_mark:</td></tr>
<tr><td>Ghost</td><td>4</td><td>0</td><td>40</td><td>:x:</td></tr>
<tr><td>Ghost</td><td>4</td><td>0</td><td>40</td><td>:x:</td></tr>
<tr><td>Wraith</td><td>4</td><td>0</td><td>45</td><td>:x:</td></tr>
<tr><td>Wraith</td><td>4</td><td>0</td><td>45</td><td>:x:</td></tr>
<tr><td>Eikthyr</td><td>6</td><td>1</td><td>120</td><td>:x:</td></tr>
<tr><td>gd_king</td><td>7</td><td>1</td><td>140</td><td>:x:</td></tr>
<tr><td>Dragon</td><td>7</td><td>2</td><td>210</td><td>:x:</td></tr>
<tr><td>Dragon</td><td>8</td><td>2</td><td>230</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>8</td><td>2</td><td>230</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>10</td><td>2</td><td>250</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>10</td><td>2</td><td>250</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>9</td><td>2</td><td>240</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>11</td><td>3</td><td>280</td><td>:heavy_check_mark:</td></tr>
</table>

## Plains
<table>
<tr><th>TargetID</th><th>Iron</th><th>Gold</th><th>Coins</th><th>Adds</th></tr>
<tr><td>Goblin</td><td>5</td><td>0</td><td>50</td><td>:x:</td></tr>
<tr><td>Goblin</td><td>5</td><td>0</td><td>50</td><td>:x:</td></tr>
<tr><td>Goblin</td><td>5</td><td>0</td><td>50</td><td>:x:</td></tr>
<tr><td>Hatchling</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Skeleton</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Lox</td><td>5</td><td>0</td><td>80</td><td>:x:</td></tr>
<tr><td>Surtling</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Draugr_Ranged</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Skeleton</td><td>6</td><td>0</td><td>45</td><td>:heavy_check_mark:</td></tr>
<tr><td>Deathsquito</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Deathsquito</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Deathsquito</td><td>8</td><td>3</td><td>50</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinBrute</td><td>0</td><td>3</td><td>80</td><td>:x:</td></tr>
<tr><td>GoblinBrute</td><td>8</td><td>3</td><td>120</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinBrute</td><td>10</td><td>3</td><td>180</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinShaman</td><td>10</td><td>3</td><td>200</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinShaman</td><td>15</td><td>4</td><td>250</td><td>:heavy_check_mark:</td></tr>
<tr><td>Ghost</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Ghost</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Wraith</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Wraith</td><td>5</td><td>0</td><td>35</td><td>:x:</td></tr>
<tr><td>Eikthyr</td><td>5</td><td>1</td><td>65</td><td>:x:</td></tr>
<tr><td>gd_king</td><td>8</td><td>1</td><td>75</td><td>:x:</td></tr>
<tr><td>gd_king</td><td>10</td><td>2</td><td>85</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>9</td><td>2</td><td>215</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>12</td><td>3</td><td>285</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>15</td><td>6</td><td>385</td><td>:heavy_check_mark:</td></tr>
<tr><td>Bonemass</td><td>15</td><td>6</td><td>385</td><td>:heavy_check_mark:</td></tr>
<tr><td>Bonemass</td><td>15</td><td>4</td><td>335</td><td>:heavy_check_mark:</td></tr>
<tr><td>Bonemass</td><td>20</td><td>5</td><td>355</td><td>:heavy_check_mark:</td></tr>
<tr><td>Bonemass</td><td>25</td><td>6</td><td>535</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>15</td><td>3</td><td>535</td><td>:x:</td></tr>
<tr><td>GoblinKing</td><td>17</td><td>3</td><td>565</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>25</td><td>4</td><td>635</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>35</td><td>8</td><td>835</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>35</td><td>8</td><td>835</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>45</td><td>10</td><td>1035</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>45</td><td>10</td><td>1035</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>55</td><td>12</td><td>1235</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>65</td><td>13</td><td>1285</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>75</td><td>15</td><td>1435</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>80</td><td>15</td><td>2035</td><td>:heavy_check_mark:</td></tr>
</table>

## Ocean
<table>
<tr><th>TargetID</th><th>Iron</th><th>Gold</th><th>Coins</th><th>Adds</th></tr>
<tr><td>Serpent</td><td>4</td><td>1</td><td>30</td><td>:x:</td></tr>
</table>

## Mistlands
<table>
<tr><th>TargetID</th><th>Iron</th><th>Gold</th><th>Coins</th><th>Adds</th></tr>
<tr><td>Ghost</td><td>5</td><td>0</td><td>40</td><td>:heavy_check_mark:</td></tr>
<tr><td>Ghost</td><td>5</td><td>0</td><td>40</td><td>:heavy_check_mark:</td></tr>
<tr><td>Wraith</td><td>5</td><td>0</td><td>40</td><td>:heavy_check_mark:</td></tr>
<tr><td>Wraith</td><td>5</td><td>0</td><td>40</td><td>:heavy_check_mark:</td></tr>
<tr><td>Eikthyr</td><td>5</td><td>0</td><td>70</td><td>:heavy_check_mark:</td></tr>
</table>

## AshLands
<table>
<tr><th>TargetID</th><th>Iron</th><th>Gold</th><th>Coins</th><th>Adds</th></tr>
<tr><td>Surtling</td><td>7</td><td>0</td><td>60</td><td>:heavy_check_mark:</td></tr>
<tr><td>Surtling</td><td>7</td><td>0</td><td>60</td><td>:heavy_check_mark:</td></tr>
<tr><td>Surtling</td><td>7</td><td>0</td><td>60</td><td>:heavy_check_mark:</td></tr>
<tr><td>Surtling</td><td>7</td><td>0</td><td>60</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>82</td><td>15</td><td>2060</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>82</td><td>15</td><td>2060</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>82</td><td>15</td><td>2060</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>82</td><td>15</td><td>2060</td><td>:heavy_check_mark:</td></tr>
<tr><td>GoblinKing</td><td>82</td><td>15</td><td>2060</td><td>:heavy_check_mark:</td></tr>
</table>

## DeepNorth
<table>
<tr><th>TargetID</th><th>Iron</th><th>Gold</th><th>Coins</th><th>Adds</th></tr>
<tr><td>Troll</td><td>6</td><td>0</td><td>50</td><td>:x:</td></tr>
<tr><td>Troll</td><td>7</td><td>1</td><td>60</td><td>:heavy_check_mark:</td></tr>
<tr><td>Troll</td><td>9</td><td>1</td><td>70</td><td>:heavy_check_mark:</td></tr>
<tr><td>Troll</td><td>11</td><td>2</td><td>100</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>81</td><td>15</td><td>2050</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>81</td><td>15</td><td>2050</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>81</td><td>15</td><td>2050</td><td>:heavy_check_mark:</td></tr>
<tr><td>Dragon</td><td>81</td><td>15</td><td>2050</td><td>:heavy_check_mark:</td></tr>
</table>

**Comment by Digitalroot** [comment]
v2.0.0-RC1 is ready for testing.  https://github.com/Digitalroot-Valheim/Digitalroot.Valheim.EpicLoot.Adventure.Bounties/files/6936292/Digitalroot.Valheim.EpicLoot.Adventure.Bounties.v2.0.0-RC1.zip

Updated Bounty Info table can be found at https://github.com/Digitalroot-Valheim/Digitalroot.Valheim.EpicLoot.Adventure.Bounties/tree/v2.0.0
